### ISSUE TITLE
bug: MavenLauncher doesn't work on macOS systems

### ISSUE THREAD

**Issue by ChrisSquare** [INRIA/spoon#3530]
Running the MavenLauncher sample as shown in the [Usage page](http://spoon.gforge.inria.fr/launcher.html) fails on OSX systems:
```
Exception in thread "main" java.io.IOException: Cannot run program "mvn": error=2, No such file or directory
```
This is thrown by `Process p = Runtime.getRuntime().exec(cmd);` in `SpoonPom.guessMavenHome()`, the code currently works on Unix type of OS but not OSX.

**Comment by monperrus** [comment]
Thanks for the bug report. The fix might be easy, would you like to give it a try in a PR? That would be awesome.

**Comment by ChrisSquare** [comment]
@monperrus Absolutely

**Comment by monperrus** [comment]
closed per #3533
